### DISCUSSION TITLE
Doubt about path structure diffs between clone and release

### DISCUSSION THREAD

**Discussion by gisms** [sepinf-inc/IPED#219] [Q&A] [answered]
Good Morning!
I have a doubt, it can be motivated by my little knowledge in the use of git, anyway. 
When I clone IPED via github, the path structure is completely modified.
But by downloading the new versions through the release menu, the old structure continues.
The idea is to maintain and continue with the old structure in the official releases and the new structure only in the development work?

**Comment by lfcnassif** [comment, marked as answer]
Hi,
Not sure if I understood the question, but the project source structure always was different from final built application structure.
After cloning from git, you must follow the build instructions on project first page to build an executable snapshot version, it will be created in target/release folder.

**Comment by gisms** [comment]
Oh My Bad!
I was forgetting the build part.
Sorry and thanks again.
